Counter to be deprecated jQuery.browser method (Judgement for MSIE)

SotaAmaya opened the pull request:
This is a counter to be deprecated jQuery.browser method.
Fixed for the judge MSIE on jQuery 1.3 later.

Commit:
Counter to be deprecated jQuery.browser method (Judgement for MSIE)

adamschwartz commented:
@SotaAmaya thanks for the PR. Rather than repeat the `userAgent` check multiple times, please refactor to something like:

```
var isIE = navigator.userAgent.match(/MSIE/)
//...
if (ieIE) {
    //...
}
//...
if (ieIE) {
    //...
}
//...
```

I'll happily merge at that point.

Commit:
Judgment MSIE to a variable

SotaAmaya committed:
MSIE version

SotaAmaya commented:
Thank you for response!
I just added two commits.
- Marshalled MSIE checking to the one variable.
- Fixed browser version check for MSIE (Instead of $.browser.version(deprecated))

How about these?

adamschwartz commented:
lgtm
